generics/messageConstants/apiResponses.js

```diff
@@ -204,5 +204,9 @@ module.exports = {
 "LIBRARY_CATEGORIES_NOT_FOUND" : "Could not found library categories",
 "URL_COULD_NOT_BE_FOUND" : "Could not found url from gcp",
 "KENDRA_SERVICE_DOWN" : "Kendra service is down",
-"SUBMISSION_LIST_FETCHED" : "Submission list fetched successfully"
+"SUBMISSION_LIST_FETCHED" : "Submission list fetched successfully",
+"CRITERIA_QUESTIONS_FETCHED_SUCCESSFULLY" : "Criteria questions fetched successfully",
+"SUCCESSFULLY_SUBMITTED_MANUAL_RATING" : "submitted the manual rating successfully",
+"RATING_SUBMISSION_FAILURE" : "Could not submit the rating",
+"FETCH_CRITERIA_QUESTIONS_FAILURE" : "rating is already done / scoring system has to be manual"
```

Comment:
@deepakr123 - Change it to - "Rating is already submitted or rating submission is not allowed."

Reply:
resolved